app/views/home/index.html.erb

```diff
@@ -87,30 +87,10 @@
           <% end %>
         </table>
 
-        <p class="lead">Available Courses</p>
-        <table class="table table-sm table-striped table-responsive-md" style="table-layout: fixed">
-          <thead>
-          <tr>
-            <th class="col-sm" scope="col">Name</th>
-            <th class="col-sm" scope="col">Description</th>
-            <th class="col-sm" scope="col" style="width: 168px"></th>
-          </tr>
-          </thead>
-          <% @open_courses.each do |course| %>
-            <tr>
-              <td>
-                <span class="overflowing"><%= course.name %></span>
-              </td>
-              <td>
-                <span class="overflowing"><%= course.description %></span>
-              </td>
-              <td class="text-right"><%= link_to "Join", join_course_path(id: course.id), method: :post, class: "btn btn-primary" %></td>
-              <td> </td>
-            </tr>
-          <% end %>
-        </table>
       <% end %>
     <% end %>
-
+  <div class="text-center mt-5">
+    <%= button_to "Browse available courses", available_courses_path, :method => "get", class: "btn btn-primary" %>
+  </div>
```

Comment:
This button is visible, when no one is logged in. Please show it only when a user is logged in.

Reply:
Should this button exist, when the user is a lecturer? Because now it is displayed for all users. Please check this with the PO because the ac do not say that clearly.